docs/topics/rn-new-features.adoc

```diff
@@ -8,16 +8,15 @@
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
 .Rules for migrating from Java EE8 to Jakarta EE9
-
 {ProductShortName} rules replace Java EE8 artifact dependencies and package import statements and rename XML schema namespaces, prefixed properties, and bootstrapping files for Jakarta EE9.
 
-Most rules contain quick fixes to automate source code changes when using an {ProductShortName} IDE plugin or extension for Red Hat CodeReady Studio, Eclipse, Eclipse Che, IntelliJ IDEA, or Microsoft Visual Studio Code.
+Most rules contain quick fixes to automate source code changes when using an {ProductShortName} IDE plugin or extension for Red Hat CodeReady Studio, Eclipse, Eclipse Che, or Microsoft Visual Studio Code.
 
 To execute the rules, use the target `jakarta-ee`.
 
 .OpenRewrite recipe support (Technology Preview)
 
-link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. OpenRewrite recipes can now be executed using the {ProductShortName} link:{ProductDocUserGuideURL}[_CLI Guide_].
+link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. OpenRewrite recipes can now be executed using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
 
 The first OpenRewrite recipe, which is shipped with {ProductShortName} 5.2.1, contains an OpenRewrite recipe that renames imported `javax` packages to their `jakarta` equivalents. We plan to include additional OpenRewrite recipes in future {ProductShortName} releases.
```

Comment:
First sentence mentions "OpenRewrite recipe" twice: "The first OpenWrite recipe ... contains an OpenRewrite recipe". Rewrite to eliminate the repetition so that the flow is more logical and less tautological.

"We plan to ..." does not belong in release notes. Why is it here?

Reply:
My goal was to make it clear that OpenRewrite wasn't just for `javax` -> `jakarta`, since that was not clear enough previously. I've removed the reference to the future. In previous companies, hints about the future were encouraged in documentation. Clearly, that's the not the Red Hat way.

Reply:
It's not that it's "not the Red Hat way". It's a legal problem. If you promise something and do not deliver, that can be a legal loophole. In theory, you could have a customer say, "I'm not paying for 'x' because you did not deliver more OpenRewrite recipes." THAT is why many companies avoid making promises or hints about the future.